### ISSUE TITLE
PWG diff

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-devanagari#42]
Should be ideally NULL

```
23689c23689
< <div n="1">— 2) {%das Zeichen für den nasalen Nachklang%} ({#\~#}): {#niranunAsika, sAnunAsika#} 
---
> <div n="1">— 2) {%das Zeichen für den nasalen Nachklang%} ({#~\#}): {#niranunAsika, sAnunAsika#} 
```

**Comment by gasyoun** [comment]
The root cause is in csl-orig/v02/pwg/pwg.txt line 13639. Fix submitted via [csl-orig PR#2864](https://github.com/sanskrit-lexicon/csl-orig/pull/2864) — once merged and the server regenerates csl-devanagari/v02/pwg/pwg.txt, this will be null.

**Comment by gasyoun** [comment]
**H1523 re-check + queue**

Earlier note that [csl-orig#2864](https://github.com/sanskrit-lexicon/csl-orig/pull/2864) fixed this was **incorrect** — current `pwg.txt` line 13639 still has `({#\~#})`.

Queued source fix (local monthly batch, not pushed to csl-orig):
```
({#\~#}) → ({#~\#})
```
- Path: `csl-corrections/batch_pending/dictionaries/pwg/change_pwg_1.txt`
- `updateByLine` applied 1 change, no UTF-8 BOM
- Closes this display diff once batch lands + csl-devanagari regenerates

**Comment by Andhrabharati** [comment]
@gasyoun 

You (or Cluade) just lookled at the isolated `anunAsika` sign here; there are 100 places having `\~` in the whole of PWG running txt!

And what prompted the correction, `({#\~#}) → ({#~\#})`?

This is against the rule that was mentioned by you yourself, at PR#2864--

> **(anudatta must precede chandrabindu, matching SLP1 convention)**

Here is a sample page from the SLP1 specification document--

<img width="542" height="293" alt="Image" src="https://github.com/user-attachments/assets/52a61597-0371-4958-a4a6-8117874aba00" />

**Comment by gasyoun** [comment]
<details>
<summary>🤖 Re-check after maintainer feedback (2026-07-28)</summary>

**Not close-ready** as a blanket PWG `\~` rewrite — agreeing with @Andhrabharati.

**Evidence (local `csl-orig/v02/pwg/pwg.txt` tip):**
- Line 13639 (the original diff target): `{#\~#}` for *nasalen Nachklang* — matches the **desired** side of the issue’s unified diff (`{#\~#}`, not `{#~\#}`).
- Residual reversed order `{#~\#}` / `~\` = **0** occurrences.
- Intentional anunasika SLP1 `{#\~#}` / `\~` ≈ **97** occurrences — bulk rewrite would corrupt data.

**Conclusion:** the single reported order typo is already correct on tip. Issue can be closed **only** if maintainers treat scope as that one line; any broader “PWG accent / anunasika encoding” programme should be a new issue, not a bulk fix of the 97 `\~` sites.

</details>
